tools/install.sh

```diff
@@ -122,15 +128,24 @@ read -e -p "Group of the user starting the ProActive service: " -i "proactive" G
 
 GROUP=$(echo $GROUP | xargs)
 
-if confirm "Do you want to create this user? [Y/n] "; then
-    if [ "$GROUP" == "$USER" ]; then
-        useradd $USER -d $PA_ROOT
+id -u "$USER" > /dev/null
+
+if (( $? != 0 )); then
+    echo "The user $USER does not exist. The installation script must create this user."
+    if confirm "Proceed ? [Y/n] "; then
+        if [ "$GROUP" == "$USER" ]; then
+            useradd $USER -d $PA_ROOT
+        else
+            useradd $USER -d $PA_ROOT -g $GROUP
+        fi
+        passwd $USER;
     else
-        useradd $USER -d $PA_ROOT -g $GROUP
+        exit 1
     fi
-    passwd $USER;
+
 fi
 
+chown $USER:$GROUP $PA_ROOT
 chown -R $USER:$GROUP $PA_ROOT/$CURRENT_PA_DIR
```

Comment:
is `CURRENT_PA_DIR` a typo ? You declare `CURRENT_PADIR` at the beginning of the file.